clients/githubrepo/graphql.go

```diff
@@ -396,12 +397,14 @@ func commitsFrom(data *graphqlData, repoOwner, repoName string) ([]clients.Commi
 				string(pr.Repository.Name) != repoName {
 				continue
 			}
+			openedByBot := strings.HasPrefix(string(pr.Author.ResourcePath), "/apps/")
```

Comment:
Can you leave a comment about what it looks like for common bots like, dependabot, renovate?

Reply:
It's usually just `/apps/<app name>`. Left a comment.